docs/quickstart.md

```diff
@@ -2,198 +2,441 @@
 hide:
   - navigation
 ---
+## __Overview__
 
-This tutorial will introduce the main NeMoS functionalities. This is intended for users that are 
-already familiar with the GLM framework but want to learn how to interact with the NeMoS API. 
-If you have used [scikit-learn](https://scikit-learn.org/stable/) before, we are compatible with the [estimator API](https://scikit-learn.org/stable/modules/generated/sklearn.base.BaseEstimator.html), so the quickstart should look 
-familiar.
+NeMoS is a neural modeling software package designed to model neural spiking activity and other time-series data
+ powered by [JAX](https://jax.readthedocs.io/en/latest/). 
 
-In the following sessions, you will learn:
+At its core, NeMoS consists of two primary modules: the __`glm`__ and the __`basis`__ module.
 
-1. [How to define and fit a GLM model.](#basic-model-fitting)
-2. [What are the GLM input arguments.](#model-arguments)
-3. [How to use NeMoS with `pynapple` for pre-processing.](#pre-processing-with-pynapple)
-4. [How to use NeMoS with `scikit-learn` for pipelines and cross-validation.](#compatibility-with-scikit-learn)
+The **`glm`** module implements a Generalized Linear Model (GLM) to map features to neural activity, such as 
+spike counts or calcium transients. It supports learning GLM weights, evaluating model performance, and exploring 
+model behavior on new inputs.
 
-Each of these sections can be run independently of the others.
 
-### Basic Model Fitting
+The **`basis`** module focuses on designing model features (inputs) for the GLM. 
+It includes a suite of composable feature constructors that accept time-series data, allowing users to model a wide 
+range of observed variables—such as stimuli, head direction, position, or spike counts— as inputs to the GLM.
 
-Defining and fitting a NeMoS GLM model is straightforward:
+
+## __Generalized Linear Model__
+
+NeMoS provides two implementations of the GLM: one for fitting a single neuron, and one for fitting a  neural population simultaneously. 
+
+### __Single Neuron GLM__
+
+You can define a single neuron GLM by instantiating an `GLM` object from the `nemos.glm` module.
```

Comment:
we might not want to write out the module in the text, in case we end up doing a refactor / changing the API later (it can be hard to catch these). (the example is fine because we'll be using doctests to check them)